.astyleignore

```diff
@@ -22,4 +22,5 @@ TESTS/mbed_hal/trng/pithy
 targets
 components/802.15.4_RF
 components/wifi
+components/TARGET_PSA/TARGET_TFM
```

Comment:
@ARMmbed/mbed-os-maintainers Heads up, another large block of code that's about to be ignored by astyle

Might need to talk with techleads about why components seems to be special in regards to code styling.

Reply:
Everything under `TARGET_TFM` is imported from an external source, so it should be ignored by astyle as I see it.

Reply:
LGTM